lib/Doctrine/DBAL/Driver/OCI8/OCI8Statement.php

```diff
@@ -72,6 +72,11 @@ class OCI8Statement implements \IteratorAggregate, Statement
     protected $_paramMap = array();
 
     /**
+     * @var array
+     */
+    protected $_bindings = array();
```

Comment:
please do not introduce a proptected member here, unless completely necessary. Otherwise we will have to keep BC for that afterwards. Also please do not use the old underscore prefix naming here. `private $bindings` is fine.